pkg/apis/app/v1alpha1/kogitoinfra_types.go

```diff
@@ -56,7 +74,7 @@ type KogitoInfraSpec struct {
 type KogitoInfraStatus struct {
 	Condition  KogitoInfraCondition            `json:"condition,omitempty"`
 	Infinispan InfinispanInstallStatus         `json:"infinispan,omitempty"`
-	Kafka      InfraComponentInstallStatusType `json:"kafka,omitempty"`
+	Kafka      KafkaInstallStatus              `json:"kafka,omitempty"`
```

Comment:
We should remove any mention to implementation from our interfaces. The status should inform the user if the `KogitoInfra` resource was able to fetch the given third party resource and use that information to configure the linked KogitoRuntimes.

Reply:
done